CasperSleep/nightshade
# 122 use yaml for data (part 1)

First part of #122

This PR setsup gulp task to read and parse yaml files. The data
is passed to the templates at compilation. 

Subsequent PRs will address
- adding correct data to yaml content
- cleaning up completed patterns to build from yaml file
### Test
1. Run `npm prune`. Run `npm install`. 
2. Run `gulp`
3. Home page content is coming from yaml file. To try for yourself open `content/base.yml`. Edit the content. Page should reload ✨
